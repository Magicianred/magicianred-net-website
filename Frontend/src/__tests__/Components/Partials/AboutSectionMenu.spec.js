import React from 'react';
import { create } from 'react-test-renderer';

import AboutSectionMenu from './../../../Components/Partials/AboutSectionMenu'

describe("Show the About Section Menu", () => {

    describe("Snapshots", () => {
    
        it("Matches the snapshot (with default params)", () => {
            const instance = create(<AboutSectionMenu />);
            expect(instance.toJSON()).toMatchSnapshot();
        });
    
    });

    xit("Show About Section Menu", () => {

    });
});