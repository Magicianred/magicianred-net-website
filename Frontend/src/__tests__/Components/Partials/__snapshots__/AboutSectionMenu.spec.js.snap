// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Show the About Section Menu Snapshots Matches the snapshot (with default params) 1`] = `
<header
  className="partial about menu"
  style={
    Object {
      "borderBottom": "solid thin black",
      "borderTop": "solid thin black",
    }
  }
>
  <nav
    className="justify-content-center navbar navbar-expand navbar-tabs"
    fill="true"
  >
    <button
      aria-controls="basic-navbar-nav"
      aria-label="Toggle navigation"
      className="navbar-toggler collapsed"
      onClick={[Function]}
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      aria-expanded={null}
      className="navbar-collapse collapse"
      id="basic-navbar-nav"
    >
      <div
        className="mr-auto navbar-nav"
        onKeyDown={[Function]}
      >
        <a
          className="nav-link"
          data-rb-event-key="/about"
          href="/about"
          onClick={[Function]}
          onKeyDown={[Function]}
        >
          About Us
        </a>
        <a
          className="nav-link"
          data-rb-event-key="/about/whoare"
          href="/about/whoare"
          onClick={[Function]}
          onKeyDown={[Function]}
        >
          Who Are
        </a>
        <a
          className="nav-link"
          data-rb-event-key="/about/whereare"
          href="/about/whereare"
          onClick={[Function]}
          onKeyDown={[Function]}
        >
          Where Are
        </a>
      </div>
    </div>
  </nav>
</header>
`;
